bundles/af-core/src/test/resources/form/formcontainer/test-page-content.json

```diff
@@ -94,7 +94,7 @@
       ],
       "fd:version": "2.1",
       "schemaType": "none",
-      "specVersion": "0.14.2",
+      "specVersion": "0.15.2",
```

Comment:
Why did you make this change ?

Reply:
Please don't make any changes in the file which is not related to this PR

Reply:
If I am not updating this version, 3 tests are failing FormContainerImplTest.testJSONExportWithoutSubmissionAttribute
FormContainerImplTest.testJSONExportWithSubmissionHeader
FormContainerImplTest.testJSONExportWithSubmissionAttribute